### ISSUE TITLE
getDrivingDirection

### ISSUE THREAD

**Issue by PHansen940** [jtsage/FS22_simpleInspector#38]
SimpleInspector.lua:374: attempt to call method 'getDrivingDirection' (a nil value)

**Comment by PHansen940** [comment]
[log.txt](https://github.com/jtsage/FS22_simpleInspector/files/14809041/log.txt)

**Comment by jtsage** [comment]
gross.  I suspect there is an odd vehicle type in your mod list that doesn't implement that function.  I'll add a trap around it to deal with it, expect the modhub version to be very slow to update though.

**Comment by PHansen940** [comment]
Is there anything I can provide you with information wise?

**Comment by jtsage** [comment]
nah, but wanna give the latest release on github a go?  in theory, it should have fixed it, but i'm stuck as far as testing right now.

https://github.com/jtsage/FS22_simpleInspector/releases/tag/v1.0.2.5

**Comment by PHansen940** [comment]
Seems to do the job!

And props for reacting and implementing the fix in such short amount of time. Thanks!
